numpy/core/_add_newdocs.py

```diff
@@ -1600,11 +1600,13 @@
 
     Values are generated within the half-open interval ``[start, stop)``
     (in other words, the interval including `start` but excluding `stop`).
-    For integer arguments the function is equivalent to the Python built-in
-    `range` function, but returns an ndarray rather than a list.
+    For integer arguments the function is roughly equivalent to the Python
+    built-in :py:class:`range`, but returns an ndarray rather than a list.
```

Comment:
This should be updated as well, since `range` now returns a `range` instance, not a list as it used to in older Python versions